AGGREGATE.md

```diff
@@ -399,6 +398,38 @@ destination site.
 Note: The storage limits for event-level and aggregatable reports are enforced
 independently of each other.
 
+### Hide the true number of attribution reports
+
+The presence or absence of an attribution report leaks some potentially
+sensitive cross-site data in the current design. Therefore, revealing the total
+count of reports to the reporting origin could leak something sensitive as well
+(imagine if the reporting origin only ever registered a conversion or impression
+for a single user).
+
+To hide the true number of reports, the browser will add noise to the number of
+reports by randomly sending noisy null reports for some fraction of trigger
+registrations. Null reports will not check or affect rate limits.
+
+As a lot of the cross site information is embedded in the source registration
+time, trigger registration will accept an optional field
+`aggregatable_source_registration_time` to allow developers to specify whether
+to omit or include `source_registration_time` field within `shared_info` in the
+aggregatable report. The null report rate will need to be higher if
+`source_registration_time` is present in the aggregatable report.
+
+```jsonc
+{
+   ...,
+   "aggregatable_trigger_data": ...,
+   "aggregatable_values": ...,
+   "aggregatable_source_registration_time": "omit" // or "include", defaults to "omit" if not present
```

Comment:
Do we think we'll actually have other options here in addition to `omit` and `include`? If not, should we use a boolean field instead?

Reply:
I don't feel too strongly, but I considered a non-boolean enum in case we wanted to extend this to something like "include_coarse" or some variants along those lines which included `source_registration_time` but rounded to a week rather than a day.

Reply:
Using a string is fine, then, but I think it should be `exclude` rather than `omit`.

Reply:
Yep, that's fine.